CHANGELOG.md

```diff
@@ -1,3 +1,149 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
```

Comment:
We should include #2863 here as well.

Maybe "Improve random-access read performance of root volume on some devices"?

Reply:
Whoops - did have that but thought it was part of Erikson's work with rpm2img as well